projects/go-lib/src/lib/components/go-badge/go-badge.component.html

```diff
@@ -0,0 +1,6 @@
+<span class="go-badge-outer-container">
+  <ng-content></ng-content>
+  <span class="go-badge-container">
+    <span class="go-badge" [ngClass]="badgeClasses()">{{ displayData ? badgeData : ''}}</span>
```

Comment:
Should we consider not using `ngClass` here because we can probably assume that this is not going to change outside the scope of this component. I think we could make use of a local object that holds this value and then make use of `onChanges` to reset these values. That way we're only reevaluating these classes when changes to the inputs happen. Thoughts?